pkg/plugin/plugin.go

```diff
@@ -92,7 +92,7 @@ type Metadata struct {
 	// Note that command is not executed in a shell. To do so, we suggest
 	// pointing the command to a shell script.
 	//
-	// DEPRECATED: Use PlatformCommand instead. Remove in Helm 4.
+	// DEPRECATED: Use PlatformCommand instead
```

Comment:
Is this going to be removed in Helm 4 or Helm 5?

Reply:
I think removing is more likely to be done in a future plugin version (rather than Helm version).